examples/correlation_alignment_example.py

```diff
@@ -85,27 +87,20 @@
             select_pathway(data.C.mean(indirect), signal_pathway)
         )
         data.setaxis("t2", lambda x: x - best_shift).register_axis({"t2": 0})
-        fl.image(data, ax=ax_list[3])
-        ax_list[3].set_title("Hermitian Test (t)")
-        data.ft("t2")
         data.ft("t2")
         data *= mysgn
         fl.image(data, ax=ax_list[1])
-        ax_list[1].set_title("Phase Corrected and Centered")
-        data *= mysgn
+        ax_list[1].set_title("Phased and Centered")
```

Comment:
you ant to remove the lines having to do with mysgn, I agree, but I don't understand why the plot title is changed.